packages/orbit-components/src/AirportIllustration/AirportIllustration.stories.tsx

```diff
@@ -3,6 +3,7 @@ import { select, text } from "@storybook/addon-knobs";
 
 // @ts-expect-error TODO
 import { NAMES } from "./consts.mts";
```

Comment:
Removing the comment triggers the error
`An import path can only end with a '.mts' extension when 'allowImportingTsExtensions' is enabled.` but then why don't we just enable this rule?

Reply:
Good point. I think when I was adding mts -> typescript did not support yet that. [It appeared quite recently ](https://github.com/microsoft/TypeScript/issues/53316) :)